doc/changelog.md

```diff
@@ -375,42 +450,23 @@
   * setting async runtime tokens and values to be errors
   * deallocating live tokens and values
 
-* Fix an issue when no qubit number was specified for the `qinst` primitive. The primitive now
-  correctly deduces the number of qubits when no gate parameters are present. This change is not
-  user facing.
-  [(#496)](https://github.com/PennyLaneAI/catalyst/pull/496)
-
 * Fix the scatter operation lowering when `updatedWindowsDim` is empty.
```

Comment:
Do we know how this could have manifested for the user?

Reply:
Good question, cc @rmoyard

Reply:
Possible for some indexing, but the main case is not covered because we removed the ability to differentiate with Jax when indexing. See https://github.com/PennyLaneAI/catalyst/pull/411#discussion_r1462447226